src/downshift.js

```diff
@@ -218,9 +218,12 @@ class Downshift extends Component {
   }
 
   scrollHighlightedItemIntoView = () => {
-    const node = this.getItemNodeFromIndex(this.getState().highlightedIndex)
-    const rootNode = this._rootNode
-    scrollIntoView(node, rootNode)
+    /* istanbul ignore else (react-native) */
+    if (preval`module.exports = process.env.BUILD_REACT_NATIVE !== 'true'`) {
```

Comment:
i dont think `preval` is actually needed here, rollup should handle this just fine with `rollup-plugin-replace`

Reply:
I like that idea. Let's add a feature to `kcd-scripts` and then upgrade this to use that feature.

I think we'll want to configure that somehow though (CLI args?) because there are some `process.env.*` variables that we want to leave as-is.